frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -68,97 +68,39 @@ interface DisplayProps {
     element: (props: any) => JSX.Element | null
     icon: (props: any) => JSX.Element | null
     viewText: string
-    link: (item: DashboardItemType) => string
+}
+
+const displayMapItem = (className: string, element: any, icon: any, viewText: string): DisplayProps => {
+    return { className, element, icon, viewText }
+}
+
+const displayMapItemLink = ({ id }: DashboardItemType): string => {
+    return `/insights/${id}`
+}
+
+export const displayHistoryItemLink = ({ id, dashboard, filters }: DashboardItemType): string => {
```

Comment:
It was a bit of a pain to reroute the Insights History with the new `insights/${id}` links so I'm leaving it as it was originally (for now) to maintain functionality.